style.css

```diff
@@ -0,0 +1,226 @@
+body {
+  background-color: #eee;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  min-height: 75px;
+  padding: 0 20px;
+  display: inline-block;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #eee;
+}
+
+nav {
+  display: inline-block;
+  margin-top: 10px;
+  width: 50%;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  margin-top: 15px;
+  width: 49.5%;
+  position: relative;
+}
+
+.logo > img {
+  width: 100%;
+  height: 100%;
+  max-width: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+ul {
+  list-style-type: disc;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding-inline-start: 40px;
+}
+
+li {
+  display: inline;
+  margin-left: 20px;
+  list-style: none;
+  text-align: -webkit-match-parent;
+}
+
+a {
```

Comment:
Kindly use classes, id for the name of your tags instead of direct tags because it helps you point out specifics of your code